src/acpi/method/method.c

```diff
@@ -255,7 +255,7 @@
  */
```

Comment:
General Commit message comment to indicate clean up/spell correction of some existing variable names.